@sridhargaddam please amend the git message:
```
[submariner]$ make gitlint
gitlint --commits origin/devel..HEAD

Commit a3a996c3c2:
4: B1 Line exceeds max length (102>80): "[commit](https://github.com/submariner-io/submariner/commit/499f0373f11dd5f6682be5f97a928f13f68a1bfd),"
make: *** [/opt/shipyard/Makefile.inc:79: gitlint] Error 1
```

--- a/pkg/netlink/netlink.go
+++ b/pkg/netlink/netlink.go
@@ -82,7 +82,7 @@ func (n *netlinkType) AddrAdd(link netlink.Link, addr *netlink.Addr) error {
 }
 
 func (n *netlinkType) NeighAppend(neigh *netlink.Neigh) error {
-	return netlink.NeighAdd(neigh)
+	return netlink.NeighAppend(neigh)
 }
 
 func (n *netlinkType) NeighDel(neigh *netlink.Neigh) error {

